spec/models/miq_server/environment_manager_spec.rb

```diff
@@ -1,15 +1,21 @@
 require 'socket'
 
 RSpec.describe "Server Environment Management" do
-  let(:mac_address) { 'a:1:b:2:c:3:d:4' }
-  let(:hostname) { Socket.gethostname }
-  let(:loopback) { '127.0.0.1' }
```

Comment:
These were only used in [this context](https://github.com/ManageIQ/manageiq/pull/20992/files#diff-b1bd389cbc64d6ee1689dbfce98e7387a6d40046144853b6c31b6e8c8ba1e272L8) so I just moved them down